tests/Application/Actions/Hooks/StripePaymentsUpdateTest.php

```diff
@@ -644,4 +630,33 @@ public function setChargeIdOnDonation(Donation $donation, string $chargeId): voi
             0,
         );
     }
+
+    /**
+     * @return array<string, mixed>
+     */
+    private function getWebhookData(string $mockName): array
+    {
+        /** @var array<string, mixed> $mock */
+        $mock = json_decode(
+            $this->getStripeHookMock($mockName),
+            associative: true,
+            flags: \JSON_THROW_ON_ERROR
+        );
+
+        return $mock;
+    }
+
+    private function sendWebhook(array $data): ResponseInterface
```

Comment:
We could definitely use this for a few additional tests too, but I ran out of time today.